_shows/18_19/when_i_watch_you.md

```diff
@@ -0,0 +1,61 @@
+---
+title: When I Watch You, I Like to Leave the Sound Off
+playwright: Cameron Brett
+student_written: true
+period: Spring
+season: Fringe
+season_sort: 280
```

Comment:
290 (#1386)

Reply:
god dammit why can i not do season sorts